api_client/python/timesketch_api_client/importer.py

```diff
@@ -199,6 +199,9 @@ def _upload_data_buffer(self, end_stream):
             'prepare.'.format(time.time() - start_time))
 
         response = self._sketch.api.session.post(self._resource_url, data=data)
+        # To prevent unexpected errors with connection refusal adding a quick
+        # sleep.
```

Comment:
Can you add a TODO here to investigate the underlying issue and get rid of this sleep when ready?

Reply:
done